### DISCUSSION TITLE
OpenThread support

### DISCUSSION THREAD

**Discussion by vgolovanov** [espressif/arduino-esp32#6328] [General]
Any plans to implement OpenThread component in Arduino Core released in IDF 4.4 ?

**Comment by pedrominatel** [comment]
Hello!

Do you have any applications for that now? This support will be probably added in the future, but we don't have it in our roadmap right now.

**Reply by vgolovanov** [reply]
Hi. 

Matter, DALI, IPBLIS, OCF.

All of them using Thread.

**Comment by VojtechBartoska** [comment]
This topic is waiting for ESP32-H2 SoC support. Keep it open and we can talk about this later.

**Comment by EwoutBergsma** [comment]
Is it later yet? :)

**Reply by pedrominatel** [reply]
Thread will be officially supported only on ESP-IDF v5.1 release. We can't support it on Arduino before moving to ESP-IDF 5.1.

**Reply by EwoutBergsma** [reply]
Thanks for your reply, highly appreciated. 

I noticed the following:
https://docs.espressif.com/projects/esp-idf/en/stable/esp32/get-started/

Which means v5.1 (stable) has been released, right?

What is the expected time-line for this to become compatible with the Arduino IDE?

Not meaning to be rude, just hugely interested.

**Reply by me-no-dev** [reply]
We already have a branch that is using ESP-IDF v5.1 but we are updating and changing many things, so final release will take some time. We plan to have an RC/Alpha available through board manager very soon, so people can start playing with the new chips and features.
